### PR TITLE
Correcting few typos and adding two images in the HotRegion documentation page

### PR DESCRIPTION
**Pull request by serenavinciguerra89**

### CODE REVIEW

File: docs/source/global_surface_emission.ipynb [hunk 1]
```diff
@@ -992,7 +992,7 @@
...
+    "You now need to reinstalled the package after replacing the contents of ``xpsi/surface_radiation_field/local_variables.pyx`` with the exact contents of ``xpsi/surface_radiation_field/archive/local_variables/tutorial_spot_and_ring.pyx``. The ``local_variables.pyx`` extension module must transform some set of *global* variables into *local* variables at the spacetime event defined by the intersection of a ray with the stellar surface. A vector of local variables is then passed to the ``xpsi/surface_radiation_field/hot.pyx`` module for evaluation of the specific intensity of radiation, w.r.t a local comoving surface frame, that after Lorentz transformation is transported along the ray to the image plane.\n",
```
Comment (ThomasEdwardRiley) [inline docs/source/global_surface_emission.ipynb:995]: could you also fix my typo "reinstalled" please?

File: docs/source/hotregion.rst [hunk 2]
```diff
@@ -8,6 +8,20 @@ HotRegion
...
+.. figure:: images/super_cede_omit.png
+   :width: 600
+
+Representation of the class instantiation options for the parameters `omit` and `cede` described below. 
+The image also shows the prefixes used to describe the parameters within a HotRegion object.
+
+.. figure:: images/HotRegion_parameters.png
+   :width: 600
+
+Representation of the parameters describing the HotRegion object, when `omit` or `cede` are set to ``True``.
```
Comment (ThomasEdwardRiley) [inline docs/source/hotregion.rst:20]: `Representation of a ``HotRegion`` instance configuration, when ``omit`` or ``cede`` are set to ``True``.
 A description of these parameters is also presented below. 
 If ``is_antiphased = True``, the parameter ``phase_shift`` is added to :math:`\pi` radians.
 For example, if ``phase_shift = 0``, the omission or the superseding region (respectively on the left and right side of the image) is in anti-phase with the observer meridian (i.e., its azimuth is :math:`\pi`).`

File: docs/source/hotregion.rst [hunk 3]
```diff
@@ -8,6 +8,20 @@ HotRegion
 Instances of :class:`~.HotRegion.HotRegion` are objects representing radiatively
 intense regions of the source photosphere.
 
+.. figure:: images/super_cede_omit.png
+   :width: 600
+
+Representation of the class instantiation options for the parameters `omit` and `cede` described below.
```
Comment (ThomasEdwardRiley) [inline docs/source/hotregion.rst:14]: Representation of the ``HotRegion`` class instantiation options ``omit`` and ``cede``. The image also shows the prefixes used for the instance attributes associated with each type of region constituting a ``HotRegion`` object.

File: docs/source/hotregion.rst [hunk 4]
```diff
@@ -16,6 +30,7 @@ intense regions of the source photosphere.
               cede, concentric
     :show-inheritance:
 
+
 .. autoclass:: xpsi.HotRegion.RayError
     :show-inheritance:
```
Comment (ThomasEdwardRiley) [inline docs/source/hotregion.rst:35]: could you add the :show-inheritance: command to the class that inherit from ParameterSubspace but is not shown in the docs, like you pointed out, please?

File: docs/source/global_surface_emission.ipynb (hunk 1 above)
Comment (serenavinciguerra89) [reply docs/source/global_surface_emission.ipynb:995]: Can you double check if it went through? I pushed the change on my fit_branch and clicked for the pull request from here, but I cannot see the commit message anywhere, so I wondered if it worked...

File: docs/source/hotregion.rst (hunk 4 above)
Comment (serenavinciguerra89) [reply docs/source/hotregion.rst:35]: Sure :) let me know if I should modify something else

File: docs/source/global_surface_emission.ipynb (hunk 1 above)
Comment (ThomasEdwardRiley) [reply docs/source/global_surface_emission.ipynb:995]: yes, it worked, thanks!

File: docs/source/hotregion.rst (hunk 4 above)
Comment (serenavinciguerra89) [reply docs/source/hotregion.rst:35]: I think I have done that too now, but let me know if something is still missing

File: docs/source/hotregion.rst (hunk 3 above)
Comment (ThomasEdwardRiley) [reply docs/source/hotregion.rst:14]: hi @serenavinciguerra89 , I made some edits to your caption. Can you please check and push the edits? Thanks!

File: docs/source/hotregion.rst (hunk 2 above)
Comment (ThomasEdwardRiley) [reply docs/source/hotregion.rst:20]: hi @serenavinciguerra89 , I made some edits to your caption. Can you please check and push the edits?

**Comment by ThomasEdwardRiley** [comment]
hi @serenavinciguerra89 , have you compiled the doc pages to see if they compile successfully with the edits? I can also do this by checking out your branch, but maybe I do not have to?

**Comment by serenavinciguerra89** [comment]
I only compiled the pages with the changes I made...how do I get the
changes you added?


Il giorno gio 28 gen 2021 alle ore 12:01 Thomas E. Riley <
notifications@github.com> ha scritto:

**Comment by ThomasEdwardRiley** [comment]
> I only compiled the pages with the changes I made...how do I get the changes you added? Il giorno gio 28 gen 2021 alle ore 12:01 Thomas E. Riley < notifications@github.com> ha scritto:
> […](#)
I didn't edit the file, just commented the code I suggest.

**Comment by serenavinciguerra89** [comment]
oh I see, sorry 😅
The changes you suggested are just about the captions, right?

Il giorno gio 28 gen 2021 alle ore 13:28 Thomas E. Riley <
notifications@github.com> ha scritto:

**Comment by ThomasEdwardRiley** [comment]
> oh I see, sorry 😅 The changes you suggested are just about the captions, right? Il giorno gio 28 gen 2021 alle ore 13:28 Thomas E. Riley < notifications@github.com> ha scritto:
> […](#)

yep!

**Comment by serenavinciguerra89** [comment]
Ok, I should have added the two captions :) Let me know if I missed
anything else...

Il giorno gio 28 gen 2021 alle ore 13:33 Thomas E. Riley <
notifications@github.com> ha scritto:

**Comment by ThomasEdwardRiley** [comment]
thank you!

**Comment by ThomasEdwardRiley** [comment]
one more thing! This is actually v0.7.4 because your previous patch on this fix branch of your fork was never merged into xpsi. Can you correct this and add the setup.py change to the CHANGELOG, please?

**Comment by serenavinciguerra89** [comment]
Should be done now :)

Il giorno gio 28 gen 2021 alle ore 14:39 Thomas E. Riley <
notifications@github.com> ha scritto:

**Pull request merged**